packages/edit-navigation/src/components/layout/index.js

```diff
@@ -51,6 +59,9 @@ export default function Layout( { blockEditorSettings } ) {
 
 	useMenuNotifications( selectedMenuId );
 
+	const hasMenus = menus?.length;
```

Comment:
Nit: explicitly cast it to boolean to match the same code in header.
```suggestion
	const hasMenus = menus?.length > 0;
```

Reply:
Yep, that was an oversight. 2f04009 makes it a bool.